spec/acceptance_cn_spec.rb

```diff
@@ -2,53 +2,21 @@
 
 describe "CN holidays" do
   %w(
-    2014-01-01
```

Comment:
Why remove historical holidays? I'd say keep them! The holidays of the past still matter.